vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -87,3 +87,4 @@ packages:
          - python3-netaddr
          - virt-install
          - network-scripts
+         - firewalld
```

Comment:
why is this needed?

Reply:
there is a [task](https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/firewall/tasks/firewalld.yaml) that involves firewalld service and in CentOS8 firewalld isn't preinstalled

Reply:
Aha. Ok. I guess it comes pre-installed with CentOS7

Reply:
When you do a local minimal install firewalld is pre-installed, but @maelk confirmed that in CI we use the cloud images, which don't seem to include it.

It seems that we also need to restart libvirtd in this case, so we may need to add a libvirtd restart task to ensure things are correctly configured when firewalld is installed/enabled after libvirtd has already started.

I tested on a CI VM and restarted both libvirtd then firewalld after a `make` failed with a timeout, but I think the important part was the libvirtd restart.

Reply:
I added the restarts, let's see if this helps.